CMake/Modules/python/PythonScripts.cmake

```diff
@@ -24,9 +24,21 @@ macro(install_python_lib package_name package_dir)
         message(STATUS "-- cd ${Flow123d_SOURCE_DIR}/CMake/Modules/python && pip install --target=${PYTHON_3RD_PARTY} ${package_name}")
         # we start pip install from directory where setup.cfg is placed thus avoiding issue with --target usage
         # https://github.com/pypa/pip/pull/3450 and https://github.com/pypa/pip/issues/3056
+        # for now we try to install package using this fix (platform centos)
         execute_process(
             COMMAND pip install --target=${PYTHON_3RD_PARTY} ${package_name}
-            WORKING_DIRECTORY ${Flow123d_SOURCE_DIR}/CMake/Modules/python)
+            WORKING_DIRECTORY ${Flow123d_SOURCE_DIR}/CMake/Modules/python
+            RESULT_VARIABLE RETURNCODE)
```

Comment:
Its due to bug. In working directory is present special configuration file which `pip` recognize. Other solution is to copy this file to current working directory.